### PR TITLE
[WHIR-1578] Add WaitSpot resources

### PR DESCRIPTION
**Pull request by j-rods**
# Description

https://whirli.atlassian.net/browse/WHIR-1578

- Create WaitSpot endpoints, interface & transformers
- Add waitspots property to subscriptions

  
# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made my code self-documenting where possible, with the use of clear variable and function names
- [ ] I have commented the reasons for any complex or unusual code, so others can understand the context
- [ ] I have made corresponding changes to the documentation
- [ ] I have added relevant tests for any new functionality I have added
- [ ] I have updated the CHANGELOG
- [ ] My changes do not generate any new warnings, either in tooling or in the browser console

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [0.0.41] - 2021-02-29
```
Comment (karl-whirli) [inline CHANGELOG.md:8]: I don't think you add the date when simply updating the code. I think that's reserved for when you're merging develop into master

**Review by karl-whirli — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (j-rods) [reply CHANGELOG.md:8]: ah right, good to know, I can remove it. When should this get released into master?

Comment (karl-whirli) [reply CHANGELOG.md:8]: Whenever you really require it to, but yeah it does have to be a separate action to making code updates

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-## [0.0.41] - 2021-02-29
+## [0.0.41]
 ### 🚀 Features
```
Comment (karl-whirli) [inline CHANGELOG.md]: Can just have it like this:

```
## [Unreleased]
### Features
- blah blah
```

**Review by karl-whirli — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (j-rods) [reply CHANGELOG.md]: 😂  ok 1 sec

**Review by karl-whirli — APPROVED** [review]

**Pull request merged**